src/index.css

```diff
@@ -1,4 +1,294 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&family=Roboto+Slab&display=swap');
+
 * {
   box-sizing: border-box;
   padding: 0;
 }
+
+body {
+  background-color: #f5f6fa;
+  padding: 30px;
+}
+
+main {
+  background-color: #fafafa;
+  padding: 2.313rem 6.25rem 16.563rem 6.25rem;
+}
+
+.navContainer {
+  display: flex;
+  height: 5.938rem;
+  background-color: #fff;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  justify-content: space-between;
+}
+
+.navDiv {
+  display: flex;
+}
+
+.title {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  color: #0290ff;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+}
+
+.navLinks > a {
+  text-decoration: none;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.categoryLink {
+  margin: 1.125rem 41.5rem 0.688rem 2.563rem;
+  opacity: 0.5;
+  color: #121212;
+}
+
+.booksList > li {
+  list-style: none;
+  border: 1px solid #e8e8e8;
+  margin-bottom: 0.938rem;
+  border-radius: 4px;
+  background-color: #fff;
+  padding: 2rem 9.1rem 1.75rem 1.68rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.bookContainer > span {
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  margin-bottom: 0.25rem;
+}
+
+.bookContainer > h2 {
+  font-family: 'Roboto Slab', serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.bookContainer > p {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  margin-bottom: 1.313rem;
+  margin-top: 0;
+}
+
+.actionsContainer > button {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  background-color: #fff;
+  padding-top: 0.188rem;
+  padding-bottom: 0.125rem;
+}
+
+.commentsBtn {
+  padding-right: 0.938rem;
+  border: none;
+  border-right: 1px solid #e8e8e8;
+}
+
+.removeBtn {
+  padding-left: 1.063rem;
+  padding-right: 0.938rem;
+  border: none;
+  border-right: 1px solid #e8e8e8;
+}
+
+.editBtn {
+  padding-left: 1.063rem;
+  border: none;
+}
+
+.progressContainer {
+  display: flex;
+  justify-content: space-between;
+  width: 23.625rem;
+  align-items: center;
+}
```

Comment:
- [ ] Can you please increase the space a bit here as given in the design guidelines ? 😃✨ Please note that we want to get a design as given to the Zeplin as mentioned in the project requirements ✌ See the images below for more clarity 👇

Your design | Given design
---| ---|
![image](https://github.com/lorenzoworx/bookstore-react/assets/106119105/75efd595-985b-4437-873b-13a794a1eb15) | ![image](https://github.com/lorenzoworx/bookstore-react/assets/106119105/c8781124-f6eb-46dc-89dd-cbb928c64fc3)